### DISCUSSION TITLE
Facing error in installation of uc_catalog extension

### DISCUSSION THREAD

**Discussion by mrvgs** [duckdb/duckdb#13298] [Q&A]
Hey there, 

I tried to install uc_catalog 

`duckdb.sql("INSTALL uc_catalog from 'http://nightly-extensions.duckdb.org';") `

but getting this error

```
HTTPException:` HTTP Error: Failed to download extension "uc_catalog" at URL "http://nightly-extensions.duckdb.org/17d598fc44/osx_arm64/uc_catalog.duckdb_extension.gz" (HTTP 403)
```

platform  - osx_arm64
duckdb - 1.0.1-dev3773
python version - 3.11.9 

referring this documentation: https://github.com/unitycatalog/unitycatalog/blob/main/docs/integrations/unity-catalog-duckdb.md 

Can you please advice on this

**Comment by datenbank** [comment]
same on window

**Comment by carlopi** [comment]
OSX + DuckDB v1.1.3 is expected to work just doing: `INSTALL uc_catalog` or the equivalent `INSALL uc_catalog FROM core;`, so the linked documentation page might need updating.

Windows I am not sure.

**Reply by datenbank** [reply]
I think the issue is security 

INSTALL uc_catalog FROM core;
HTTP Error: Failed to download extension "uc_catalog" at URL "http://extensions.duckdb.org/v1.1.3/windows_amd64/uc_catalog.duckdb_extension.gz" (HTTP 403)
going to url
http://extensions.duckdb.org/v1.1.3/windows_amd64/uc_catalog.duckdb_extension.gz you get <Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
</Error>

**Reply by zoevannoppen** [reply]
I'm having the same issue, is this already solved?

**Reply by carlopi** [reply]
I am not sure if Windows was/is currently supported.
@samansmink?

**Reply by zoevannoppen** [reply]
Oh, I see. Any timeline when it will be?

**Comment by carlopi** [comment]
This works for released extensions, such as 1.4.0, I think it's solved.
